town.logic

```diff
@@ -1,2 +1,107 @@
 area Town:
-  room Main
\ No newline at end of file
+  room Main:
+    node Anju:
+      chest AnjuHP
+    node AnjuRoof:
+      door AnjuChimney
+    node CatsMinish:
+      door Fountain
+      door StratoMinishBack
+    node FigurineHouseDoorEast:
+      door FigurineHouseEast
+    node GormanHouseSouth:
+      door GormanHouseSouth
+    node HearthLedge:
+      door HearthLedge
+      chest HearthLedge
+    node JuliettaRoof:
+      door JuliettaChimney
+      door RomioChimneyEast
+    node LibraryRoof:
+      door LibraryRoof
+    node LibraryRoofMinish:
+      door FlipperCave
+      door LibraryMinish
+    LibraryRoof -> LibraryRoofMinish: item Cane
+    node Main:
+      chest CafeLady
+      door Bakery
+      door Borlov
+      door East
+      door FigurineHouseWest
+      door HearthNorth
+      door HearthSouth
+      door GormanHouseWest
+      door Julietta
+      door LibraryBottom
+      door MayorWest
+      door North
+      door PostOffice
+      door Rem
+      door Romio
+      door SchoolWest
+      door Simon
+      door South
+      door Stockwell
+      door Strato
+      door Swiftblade
+      door Well
+      entrance Crest
```

Comment:
We probably also need the flag for the wind crest to be able to be set here - ignoring that the rando currently sets it by default, that'd be expressed elsewhere.
```suggestion
      entrance Crest
      flag Virtual.Zeffa.TownCrest
```